setup.py

```diff
@@ -9,7 +12,7 @@
 
 setuptools.setup(
     name=PKG,
-    version='0.1.2',
+    version='1.0.0',
```

Comment:
```suggestion
    # This tag is automatically updated by bump2version
    version='1.0.0',
```

Reply:
Yes, absolutely should have this warning here, thanks for the suggestion! I'll commit it, but feel free to parse `_verison.py` here instead if it works better for you.